### PR TITLE
Allow custom jinja filters to be used for parsing

### PR DESCRIPTION
**Pull request by tedoc2000**
Added an optional "filters" argument to loader.load that allows callers to supply a dictionary of filter name -> filter methods. This allows clients to use their own custom jinja2 filters while parsing maestro templates

### CODE REVIEW

**Comment by mpetazzoni** [comment]
@tedoc2000 What's going on with your branch? Does this need to be rebased?

**Comment by tedoc2000** [comment]
Let me try again..
